### PR TITLE
Sync headers_sent

### PR DESCRIPTION
**Pull request by xepozz**
https://www.php.net/manual/en/function.headers-sent.php

### CODE REVIEW

**Comment by LolGleb** [comment]
The tests failed because the `headers_sent` function in the php source code lacks a parameter type declaration  - https://github.com/php/php-src/blob/master/ext/standard/basic_functions.stub.php#L2264
It should be done the same for stubs. Please remove the unnecessary type declarations.

File: standard/standard_4.php [hunk 1]
```diff
@@ -728,7 +728,7 @@ function header_remove(?string $name): void {}
  * @return bool headers_sent will return false if no HTTP headers
  * have already been sent or true otherwise.
  */
-function headers_sent(&$filename, &$line): bool {}
+function headers_sent(string &$filename = null, int &$line = null): bool {}
```
Comment (xepozz) [inline standard/standard_4.php:731]:
```suggestion
function headers_sent(&$filename = null, &$line = null): bool {}
```

Comment (xepozz) [reply standard/standard_4.php:731]: So can we add only the default values then?

Comment (LolGleb) [reply standard/standard_4.php]: That is correct! Tests are green now

**Pull request merged**